packages/@okta/i18n/src/properties/login.properties

```diff
@@ -817,7 +817,9 @@ oie.select.authenticators.verify.title = Verify it\'s you with an authenticator
 oie.select.authenticators.verify.subtitle = Select from the following options
 
 ## Success Redirect
-oie.success.redirect = You are being redirected
+oie.success.text.signingIn = Signing in
+oie.success.text.signingInWithAppName = Signing in to {0}
```

Comment:
we generally don't use camel case.. instead `oie.success.text.signingIn.with.appName`.. same for the other one.